### PR TITLE
Zigbee2MQTT

### PR DESCRIPTION
**Pull request by alanmgomes**
Zigbee2MQTT enhanced app

### CODE REVIEW

**Comment by dgibbs64** [comment]
Please add an icon (preferable svg) thank you

File: TarTenNinety/app.json
Comment (mvdkleijn) [inline TarTenNinety/app.json:1]: This shouldn't be in here I believe

File: zigbeemqtt/zigbee2mqtt.png
Comment (mvdkleijn) [inline zigbeemqtt/zigbee2mqtt.png:1]: This icon is way to big. Please read through https://github.com/linuxserver/Heimdall-Apps/wiki/Image-requirements

hint: 250x250 & tinypng

**Review by mvdkleijn — CHANGES_REQUESTED** [review]
Hi there, new here, working through old PRs. If you're still willing to support this PR: please update it against latest master and work through the review comments I left.

Note: I don't know why people where force-pushing code, that's a big no no IMHO.

If you no longer want this PR, feel free to close it.

**Comment by mvdkleijn** [comment]
As this is from pre-2023 and no replies were made since my initial comment, I'm closing this under the assumption this will no longer be worked on. Feel free to leave a comment so we can re-open the PR if needed.

**Pull request closed without merging**